src/tests/tests_game.go

```diff
@@ -0,0 +1,73 @@
+package tests
+
+import "quarto/game"
+import "testing"
+import "strconv"
+
+// TestDoAMove return correct state
+func TestDoAMove(t *testing.T) {
+	var state = game.DoAMove(game.State{})
+	if state.Grid[0][0] != 0 {
+		t.Errorf("Grid should be empty at first move")
```

Comment:
This is a pretty shallow test. Can't you test that the resulted state is the same as an another ?